tests/function-controller/pkg/apirule/types/apirule.go

```diff
@@ -0,0 +1,100 @@
+// Copied from:
```

Comment:
Copied is required by problems with mods? If yes I haven't a problem with that, if not please remove it.

Reply:
It's here because of problems with mods